client/src/Pages/Game.js

```diff
@@ -9,7 +10,50 @@ import Scores from '../Components/Scores/Scores';
 import Lives from '../Components/Lives/Lives';
 import TimerBar from '../Components/TimerBar/TimerBar';
 
-export default function GamePage() {
+export default function GamePage({ wordData }) {
+  const [
+    selectedWord,
+    setSelectedWord,
+  ] = useState('');
+  const [
+    correctAnswer,
+    setCorrectAnswer,
+  ] = useState('');
+  const [
+    displayAnswer,
+    setDisplayAnswer,
+  ] = useState(true);
+
```

Comment:
`displayAnswer` sounds like a function name, but it seems to be a boolean. What about something like `isAnswerDisplayed`?

Reply:
Thanks for the suggestion! :)